packages/hint-compat-api/package.json

```diff
@@ -24,6 +24,7 @@
     "npm-run-all": "^4.1.5",
     "nyc": "^13.1.0",
     "postcss": "^7.0.13",
+    "postcss-safe-parser": "^4.0.1",
```

Comment:
Why do we need `postcss` and `postcss-safe-parser` in the `compat-api` hint? Isn't all the work done in the parser?

Reply:
Yes, you are right, but since we had to add postcss to this hint, I thought we need to manage it in the same way. Maybe postcss is only useful for typing purposes in the compat-api hint. Is that correct?